src/main/java/com/cloudbees/jenkins/plugins/bitbucket/BitbucketBuildStatusNotifications.java

```diff
@@ -146,6 +146,14 @@ private static String getHash(@CheckForNull SCMRevision revision) {
         @Override
         public void onCheckout(Run<?, ?> build, SCM scm, FilePath workspace, TaskListener listener, File changelogFile,
                                SCMRevisionState pollingBaseline) throws Exception {
+
+            if (workspace.getRemote().endsWith("@script")) {
```

Comment:
This is not always `@script`, it could use another special character instead of `@`.
Use `System.getProperty(WorkspaceList.class.getName(), "@") + "script"`.